db/schema.rb

```diff
@@ -203,6 +203,7 @@
     t.date "consent_at"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.integer "language", default: 0
```

Comment:
Waarom wordt de taal opgeslagen als een integer?

Reply:
Anders zitten er heel veel redudant data in de database, beter kan je een functional dependancy als deze met een minimale key (zoals een int) op slaan.

Reply:
Ik snap opzich het punt, maar ik vind 2 letter landcodes als 'EN' of 'NL' niet perse heel redundant